reflex/components/component.py

```diff
@@ -529,7 +529,7 @@ def _exclude_props(self) -> list[str]:
         """
         return []
 
-    def _render(self, props: dict[str, Any] | None = None) -> Tag:
+    def _render_tag(self, props: dict[str, Any] | None = None) -> Tag:
```

Comment:
I think this one we want to be a public method since users may override it `render_tag`

Should we do this one in the follow up when we change the public api?